Server/Components/DropdownButton.razor

```diff
@@ -1,22 +1,18 @@
 ﻿@inject IJsInterop JsInterop
 
-<div class="@WrapperClass" @onmouseout="MouseLeft" @onmouseover="MouseEnter">
-    <button class="btn dropdown-toggle @ButtonClass @_showClass"
+<div class="@WrapperClass">
+    <button class="btn dropdown-toggle @ButtonClass"
             type="button"
-            data-bs-toggle="dropdown"
-            aria-expanded="@_isExpanded"
-            @onclick="ToggleShown">
+            data-bs-toggle="dropdown">
         @ButtonContent
     </button>
-    <ul class="dropdown-menu @DropDownMenuClass @_showClass" @onclick="ToggleShown" >
+    <ul class="dropdown-menu @DropDownMenuClass" >
         @ChildListItems
     </ul>
 </div>
 
 @code {
     private System.Timers.Timer? _collapseTimer;
-    private string? _showClass;
```

Comment:
Not sure if these warnings are still applicable since it appears that code was removed.

Reply:
I removed the unused field, if that's what you mean?